harnesses.json

```diff
@@ -0,0 +1,14 @@
+{
+  "teufelaudio/spryker": {
+    "v1.5.1": {
+      "type": "tar.gz",
+      "url": "https://github.com/teufelaudio/harness-spryker/archive/1.5.1.tar.gz",
+      "date": "2023-05-30"
+    },
+    "v1.6.0": {
+      "type": "tar.gz",
+      "url": "https://github.com/teufelaudio/harness-spryker/archive/refs/heads/task/WEBDEV-9176-migrate-spryker-harness.tar.gz",
```

Comment:
We need to change this to an actual release version path before we merge this PR.

Reply:
ok